RPTools/maptool
# Disables Token Selection buttons when the server has no Individual Views

Fixes #656


### Description of the Change

This PR disables the 4 Token Selection buttons when a client connects to a server without the Individual View policy.

The change is implemented via an additional method in the `ToolbarPanel` which can enable and disable the four buttons as a whole.

The method is called when a client connects to a server, if that is needed, and when the client either disconnects as a choice of the user or the server closes the connection.

### Tests done

- test with Individual Views: connect to a server with Individual Views enabled, and verify that the 4 buttons remain enabled
- test without Individual Views: connect to a server with Individual Views disabled, and verify that the 4 buttons are disabled
  - verify that if the client disconnects, the buttons are re-enabled
  - verify that if the server disconnects, the buttons are enabled

### Possible Drawbacks

None.

### Documentation Notes

Buttons before connecting:

![buttons_before_connecting](https://user-images.githubusercontent.com/1532349/228370034-3c1d9f2c-1fae-44df-bbc4-6084d1095625.JPG)

Buttons after connecting:

![buttons_after_connecting](https://user-images.githubusercontent.com/1532349/228370077-4c59530d-47d1-43a2-80fa-2075c006c79d.JPG)



### Release Notes

- disables Token Selection buttons when the server has no Individual Views


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/RPTools/maptool/3885)
